src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -108,30 +207,121 @@ export default class Sidewinder extends Module {
 		if (!this._badCasts.length) {
 			return
 		}
+
+		let totalPotencyLost = 0
 		// Builds a list item for each incorrect cast
 		const items = this._badCasts.map(cast => {
 			const ability = getDataBy(ACTIONS, 'id', cast.abilityId)
-			if (cast.dotsApplied.length) {
-				return <List.Item key={cast.timestamp}>
-					<List.Content>
-						<Trans id="brd.sidewinder.list.one-dot">
-							{this._createTimelineButton(cast.timestamp)} <ActionLink {...ability}/> was cast with only <StatusLink {...getDataBy(STATUSES, 'id', cast.dotsApplied[0])}/> applied.
-						</Trans>
-					</List.Content>
-				</List.Item>
+			let singleTargetItem
+			let notBothDotsItem
+
+			if (!cast.hasBothDots) {
+				if (cast.dotsApplied) {
+					notBothDotsItem =
+							<Trans id="brd.sidewinder.list.one-dot">
+								<ActionLink {...ability}/> was cast with only one DoT applied to the target {cast.targetsHit > 1 ? 'and hit ' + cast.targetsHit + ' enemies' : ''}
+							</Trans>
+
+				} else {
+					notBothDotsItem = <Trans id="brd.sidewinder.list.no-dots">
+						<ActionLink {...ability}/> was cast with no DoTs applied to the target {cast.targetsHit > 1 ? 'and hit ' + cast.targetsHit + ' enemies' : ''}
+					</Trans>
+
+				}
 			}
+			if (cast.isSingleTargetShadowbite) {
+				singleTargetItem = <Trans id="brd.sidewinder.list.single-target-shadowbite">
+					{!cast.hasBothDots ? ' and' : <ActionLink {...ability}/>}  only hit a single enemy
+				</Trans>
+			}
+
+			totalPotencyLost += cast.missedPotency
+
 			return <List.Item key={cast.timestamp}>
-				<List.Content>
-					<Trans id="brd.sidewinder.list.no-dots">
-						{this._createTimelineButton(cast.timestamp)} <ActionLink {...ability}/> was cast with no DoTs applied.
-					</Trans>
+				{this._createTimelineButton(cast.timestamp)}
+				<List.Content verticalAlign="middle">
+					{notBothDotsItem}{singleTargetItem}. <Label horizontal size="small" color="red" pointing="left"><Trans id="brd.sidewinder.list.missed-potency"><Icon name="arrow down"/>Lost {this._formatDamageNumber(cast.missedPotency)} potency</Trans></Label>
```

Comment:
Try to keep all text that's part of a single statement/phrase in one translation ID. Context/grammar/wording/symbols change between different languages, so what works in EN often will break in others. That trailing `.` likewise may be incorrect in some other languages.